aws/resource_aws_cloudwatch_event_bus_test.go

```diff
@@ -0,0 +1,184 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	events "github.com/aws/aws-sdk-go/service/cloudwatchevents"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_cloudwatch_event_bus", &resource.Sweeper{
+		Name: "aws_cloudwatch_event_bus",
+		F:    testSweepCloudWatchEventBuses,
```

Comment:
It looks like an event bus can't be deleted if it has associated rules or targets so we'll need to add

```go
Dependencies: []string{""},
```

here.